lemmy_structs/src/lib.rs

```diff
@@ -100,7 +100,7 @@ fn do_send_local_notifs(
       // Let the uniqueness handle this fail
       match UserMention::create(&conn, &user_mention_form) {
         Ok(_mention) => (),
-        Err(_e) => error!("{}", &_e),
+        Err(_e) => (),
```

Comment:
Out of curiosity, there were a lot of errors in the logs that were tagging the same people multiple times?

Reply:
I dont know if it makes sense to ignore the error, but if so, doing `let _ = UserMention::create();` would be a shorter way to ignore the error.

Reply:
yep, PG logs were flooded with it. it occurred every time someone edited their comment.